app/javascript/lib/toasts.ts

```diff
@@ -22,14 +20,3 @@ export function toast(message: string, options?: ToastifyOptions) {
     className: options?.type && TYPE_TO_CLASS_MAP[options?.type],
   }).showToast();
 }
-
-export function renderBootstrappedToasts() {
```

Comment:
This is moved to `vue_toasts.ts` (and now using `vue-toastification` rather than `toastify`).